src/tests/fixtures/strings.js

```diff
@@ -0,0 +1,54 @@
+export const strings = [
+  {
```

Comment:
I would simulate the searched string values:
`null`, `some number`, `some weird value like fucked up object` and `undefined`, but in the way, that you are invoking function without any argument.

Reply:
I mean, as an addition to thing you already assumed :)